lib/perl/Genome/ConfigSpec.t

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env genome-perl
+
+use strict;
+use warnings;
+
+use Genome;
```

Comment:
This one doesn't have the `qw()` that the other instances of `use Genome` in this PR do.  (It's a strange world if `Genome` starts exporting subroutines by default!)